public/src/inverted-index.js

```diff
@@ -92,30 +94,30 @@ class Index {
     return check;
   }
 
-/**
- * @method getIndex
- *
- * Method return index of files in indices object
- *
- * @param {string} name
- * @returns {object}
- */
+  /**
+  * @method getIndex
+  *
+  * Method return index of files in indices object
+  *
+  * @param {string} name
+  * @returns {object}
+  */
   getIndex(name) {
     if (name && typeof name === 'string') {
       return this.indices[name];
     }
     return this.indices;
   }
 
-/**
- * @method searchFeedback
- *
- * Takes terms of array and fetch result of each token.
- *
- * @param {array} termsArray
- * @param {object} file
- * @returns {object} searchResults
- */
+  /**
+  * @method searchFeedback
+  *
+  * Takes terms of array and fetch result of each token.
+  *
+  * @param {array} termsArray
+  * @param {object} file
+  * @returns {object} searchResults
+  */
   searchFeedback(termsArray, file) {
```

Comment:
### Critical

The argument you call `file` here, is actually the created Index for the file, why would you name it `file` as opposed `fileIndex`, do you realize how that can be confusing?